sdk/ml/azure-ai-ml/azure/ai/ml/entities/_builders/spark.py

```diff
@@ -12,11 +12,11 @@
 
 from marshmallow import INCLUDE, Schema
 
-from azure.ai.ml._restclient.v2022_12_01_preview.models import IdentityConfiguration
-from azure.ai.ml._restclient.v2022_12_01_preview.models import JobBase as JobBaseData
-from azure.ai.ml._restclient.v2022_12_01_preview.models import SparkJob as RestSparkJob
-from azure.ai.ml._restclient.v2022_12_01_preview.models import SparkJobEntry as RestSparkJobEntry
-from azure.ai.ml._restclient.v2022_12_01_preview.models import (
+from azure.ai.ml._restclient.v2023_02_01_preview.models import IdentityConfiguration
```

Comment:
Note: may miss some change when this is merged: 
https://github.com/Azure/azure-sdk-for-python/pull/28961

Reply:
Thanks for the note. 
Please let me know if I missed any expected changes.